### PR TITLE
adding examples for pod / configmap

### PR DESCRIPTION
**Pull request by eifrach**
Signed-off-by: Eran Ifrach <eifrach@redhat.com>

Adding Examples files for Assisted Installer deployment as pod

1. pod.yml which describes all the continers 
2. Configmap.yml which contians enviroment configurations

> note: the pod is using volume map for consistent database

### CODE REVIEW

File: examples/configmap.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
..."openshift_version": "4.9", "cpu_architecture": "x86_64", "url": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-fc.2/rhcos-4.10.0-fc.2-x86_64-live.x86_64.iso", "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-fc.2/rhcos-4.10.0-fc.2-x86_64-live-rootfs.x86_64.img", "version": "410.84.202201191157-0"}, {"openshift_version": "4.10", "cpu_architecture": "x86_64", "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.10.0-fc.4/rhcos-4.10.0-fc.4-x86_64-live.x86_64.iso", "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.10.0-fc.4/rhcos-4.10.0-fc.4-x86_64-live-rootfs.x86_64.img", "version": "410.84.202201261816-0"}, {"openshift_version": "4.10", "cpu_architecture": "arm64", "url": "https://mirror.openshift.com/pub/openshift-v4/arm64/dependencies/rhcos/pre-release/4.10.0-fc.4/rhcos-4.10.0-fc.4-aarch64-live.aarch64.iso", "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/arm64/dependencies/rhcos/pre-release/4.10.0-fc.4/rhcos-4.10.0-fc.4-aarch64-live-rootfs.aarch64.img", "version": "410.84.202201261816-0"}]'
```
Comment (tsorya) [inline examples/configmap.yml]: let remove 4.9

File: examples/configmap.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    RELEASE_IMAGES: '[{"openshift_version": "4.9", "cpu_architecture": "x86_64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-fc.3-x86_64", "version": "4.10.0-fc.3", "default": true}, {"openshift_version": "4.9", "cpu_architecture": "arm64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-fc.3-aarch64", "version": "4.10.0-fc.3"}, {"openshift_version": "4.10", "cpu_architecture": "arm64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.5-aarch64", "version": "4.10.0-rc.5"}, {"openshift_version": "4.10", "cpu_architecture": "x86_64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.5-x86_64", "version": "4.10.0-rc.5"}]'
```
Comment (tsorya) [inline examples/configmap.yml]: same

File: examples/configmap.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    POSTGRESQL_USER: admin
+    DB_HOST: 127.0.0.1
+    DB_PORT: 5432
+    DB_USER: admin
+    DB_PASS: password
+    DB_NAME: installer
+    AUTH_TYPE: "none"
+    DEPLOY_TARGET: onprem
+    SKIP_CERT_VERIFICATION: true
+    INSTALLER_RELEASE_IMAGE_OVERRIDE_UNSUPPORTED: "quay.io/openshift-release-dev/ocp-release:4.10.0-fc.3-x86_64"
```
Comment (tsorya) [inline examples/configmap.yml]: not needed anymore

File: examples/configmap.yml (hunk 2 above)
Comment (tsorya) [reply examples/configmap.yml]: RELEASE_IMAGES=[{"openshift_version": "4.10", "cpu_architecture": "arm64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.6-aarch64", "version": "4.10.0-rc.6"}, {"openshift_version": "4.10", "cpu_architecture": "x86_64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.6-x86_64", "version": "4.10.0-rc.6"}]

File: examples/configmap.yml (hunk 1 above)
Comment (tsorya) [reply examples/configmap.yml]: OS_IMAGES=[{"openshift_version": "4.10", "cpu_architecture": "x86_64", "url": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live.x86_64.iso", "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live-rootfs.x86_64.img", "version": "410.84.202201261816-0"}, {"openshift_version": "4.10", "cpu_architecture": "arm64", "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-aarch64-live.aarch64.iso", "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-aarch64-live-rootfs.aarch64.img", "version": "410.84.202201261816-0"}]
RELEASE_IMAGES=[{"openshift_version": "4.10", "cpu_architecture": "arm64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.6-aarch64", "version": "4.10.0-rc.6"}, {"openshift_version": "4.10", "cpu_architecture": "x86_64", "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-rc.6-x86_64", "version": "4.10.0-rc.6"}]

File: examples/configmap.yml (hunk 3 above)
Comment (eifrach) [reply examples/configmap.yml]: all the lines? 

15 - 18 ?

**Pull request merged**